rllib/agents/dqn/r2d2.py

```diff
@@ -22,7 +22,7 @@
         # Discount factor.
         "gamma": 0.997,
         # Train batch size (in number of single timesteps).
-        "train_batch_size": 64,
+        "train_batch_size": 2560,
```

Comment:
Huh, why this drastic change?

Before, did this setting refer to whole sequences (even though it said "single timesteps" in the comment)?

Reply:
Yes, indeed. I just checked back with our last release, using the legacy buffers, and we indeed replayed sequences rather than timesteps there. 

When I first realized this I went to replay_sequence_length * train_batch_size = 40 * 64 = 2560.
But I checked back with the last release and for some reason the replay _sequence_length was not equal to the train_batch_size. It was 32.
I changed this to 1000 in my newest commit, because going through a couple of iterations I the number of timesteps that we sample was around 1000 because we have no concept for counting batch size including padding and burn in atm.